README.md

```diff
@@ -32,6 +32,25 @@ See [design/](design/README.md).
 
 See [configuration/](configuration/README.md) for configuration details.
 
+### Debugging
+
+The main analyzer depends heavily on the SSA package. Being able to read the SSA code and visualize its graph can be very useful for debugging. In order to generate the SSA code and DOT (graphviz) source for every function in a test, run `go test levee_test.go -debug`. Results are written to the `output` directory. You can generate a PDF from the DOT source using `dot -Tpdf <file> -o "$(basename <file> .dot).pdf"`.
+
+Currently, debugging is only supported for `levee_test.go`. In order to add support for debugging in a new test, first add a debugging flag:
+```go
+var debugging bool
+
```

Comment:
It can be a future PR, but this feels cumbersome.  I think it would better if this debug plotter were something we could freely target and run independent of testing.

Reply:
I agree that this is not optimal. I'll keep this in the back of my mind and if I find a better idea, I'll make a new PR.